src/components/shared/messages/alerts.js

```diff
@@ -0,0 +1,24 @@
+import { useSnackbar } from 'notistack';
+import {useEffect, useState} from "react";
+
+ //default, success, error, warning , info
+const useNotification = () => {
+    const [conf, setConf] = useState({});
+    const { enqueueSnackbar } = useSnackbar();
+   
+    useEffect(()=>{
+        if(conf?.msg){
+            let variant = 'info';
+            if(conf.variant){
+                variant = conf.variant;
+            }
+            enqueueSnackbar(conf.msg, {
+                variant: variant,
+                autoHideDuration: 5000,
+            });
+        }
+    },[conf]);
+    return [conf, setConf];
+};
+
+export default useNotification;
```

Comment:
line at the end is missing, plus the coding style is different, and this should have triggered an error